mappings/net/minecraft/server/OperatorList.mapping

```diff
@@ -1,3 +1,3 @@
 CLASS net/minecraft/unmapped/C_hlmbwrkb net/minecraft/server/OperatorList
-	METHOD m_lpbxcmbi isOp (Lcom/mojang/authlib/GameProfile;)Z
+	METHOD m_lpbxcmbi canBypassPlayerLimit (Lcom/mojang/authlib/GameProfile;)Z
```

Comment:
This sounds odd, need confirmation.

Reply:
It's also `canBypassPlayerLimit` in mojmap

Reply:
Definitely need to look at the code myself because this sounds so restrictive compared to the previous name...

Reply:
![Screenshot_20211004-083150.png](https://user-images.githubusercontent.com/47987888/135844391-06015d7e-a50c-4a12-9b28-69a3277c3534.png)

Reply:
Guess I can't deny that, though I would've looked at the usages instead *shrugs*

Reply:
The two usages are in `PlayerManager#checkCanJoin` and `PlayerManager#addToOperators`